AmplifyPlugins/Predictions/AWSPredictionsPlugin/Liveness/Model/FaceLivenessSession+BoundingBox.swift

```diff
@@ -7,6 +7,7 @@
 
 import Foundation
 
+// swiftlint:disable identifier_name
```

Comment:
Curious as to why we are disabling it here and enabling it later.

Reply:
disabled rules need to be re-enabled again for the rules to run again.  By default, the rules get re-enabled at the end of the file again, I added it anyways at the end of the file for readability/clarity.